components/markdown/src/markdown.rs

```diff
@@ -139,7 +139,32 @@ fn fix_link(
                 resolved.permalink
             }
             Err(_) => {
-                return Err(anyhow!("Relative link {} not found.", link));
+                context.config.link_checker.internal_level.log(format!(
+                    "Dead relative link `{}` in {}",
+                    link,
+                    context.current_page_path.unwrap_or("unknown"),
+                ));
+                match context.config.link_checker.internal_level {
+                    config::LinkCheckerLevel::Error => {
+                        return Err(anyhow!(
+                            "Dead relative link `{}` in {}",
+                            link,
+                            context.current_page_path.unwrap_or("unknown"),
+                        ))
+                    }
+                    config::LinkCheckerLevel::Warn => {
+                        console::warn(
+                            format!(
+                                "{}Dead relative link `{}` in {}",
+                                config::LinkCheckerLevel::Warn.log_prefix(),
+                                link,
+                                context.current_page_path.unwrap_or("unknown"),
+                            )
+                            .as_str(),
+                        );
```

Comment:
that's duplicated code with `context.config.link_checker.internal_level.log` above no?

Reply:
Yeah, thanks, I missed that one.